.gitignore

```diff
@@ -8,8 +8,6 @@ src/gql/utils/generated.ts
 src/gql/utils/schema.graphql
 dist
 **/node_modules/*
-nibiru/
```

Comment:
does this mean we don't need the file locally to generate the protos?

Reply:
Its done via git submodules now